src/modules/i18n/locales/rm.json

```diff
@@ -391,6 +391,7 @@
     "meteoschweiz_service_link_href": "http://www.meteoschweiz.admin.ch",
     "meteoschweiz_service_link_label": "MeteoSvizra",
     "mobile_redirect": "Versiun mobila",
+    "modify_add_vertex": "Thov nevo punkto vaś te lungǎres i alosardi linia",
```

Comment:
I've updated the i18n files, and check this translation here, my tool has something else.
Can you use
`Agiuntar in nov punct per prolungar la lingia tschernida` instead?

Reply:
@pakb I have rebased with develop, so I am not sure how else can I check if the translation is updated or not.